data_structure/array.py

```diff
@@ -0,0 +1,80 @@
+class PlusOne(object):
+    # instance method
+    def plusOne(self, digits):
+        carry = 1
+        len_digits = len(digits)
```

Comment:
len_digits is used only once so it's probably not needed. Just do `for i in reversed(range(len(digits)))`